docs/ABOUT.md

```diff
@@ -1,31 +1,59 @@
-ECMAScript is the standard that defines JavaScript, the language that allows web
-pages to be dynamic. It is an interpreted language, which means that it doesn't
-need to be compiled by the programmer: instead the client (such as a web
-browser) will parse the code and turn it into code that their machine can run -
-suitable for creating dynamic websites that can run on any browser on any
-computer!
-
-In addition to use in web pages with modern web browsers, and it can also be
-executed on servers where the NodeJS platform is installed where it can be used
-for creating a web server too.
-
-"ECMAScript has grown to be one of the world’s most widely used general purpose
-programming languages.
-It is best known as the language embedded in web browsers but has also been
-widely adopted for server and embedded applications."
-— [ECMA International Language Specification](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-ecmascript-overview)
+ECMAScript is the [standard][web-ecma] that defines JavaScript, a programming 
+language that allows web pages to be dynamic. It is an interpreted language, 
+which means that it doesn't need to be compiled by the programmer: instead the 
```

Comment:
`compiled by the programmer` is little misleading IMO - the `compiler` compiles 😄. Maybe just leave it as `to be compiled`?

Reply:
Ah, I literally had that removed (because I already didn't like it before) but I guess I undo-ed too much :P. Resolved!